### ISSUE TITLE
Header checkbox disabled?

### ISSUE THREAD

**Issue by DSBhogal** [gustavnavar/Grid.Blazor#180]
### Description
On all areas of my application where I use Grid.Blazor with a checkbox column the header checkbox appears disabled. When i initially click the checkbox nothing happens. If i persistently attempt to select the header checkbox it will intermittently work. I am using v1.6.9 of Grid.Blazor.

### Screenshots
** Application Running**
![image](https://user-images.githubusercontent.com/33750091/86104086-7dc30080-bab5-11ea-9ff7-4bb5e81ee62a.png)

**Code**
![image](https://user-images.githubusercontent.com/33750091/86104205-a3500a00-bab5-11ea-8f61-6ccc71dc0247.png)

**Comment by gustavnavar** [comment]
The header checkbox is never disabled. When it has a grey background is when the grid has rows checked and unchecked at the same time.  Maybe the grey background is misleading.

If all the rows that are shown on the screen  are unchecked and the header checkbx has a grey background, there must be a checked row on any other page.

**Comment by DSBhogal** [comment]
When you say on any other page do you mean on the pagination of the grid? because I perform a query when getting my grid rows to check for checked rows and that returns nothing. If you mean if rows on other grids on separate pages are checked/unchecked then how does one stop that from happening?

**Comment by DSBhogal** [comment]
I don't think this is an checked/unchecked issue, after fiddling with the CSS on the header checkbox if i remove:

1. **opacity** from **.grid-header-checkbox-input** within **gridblazor.min.css**
2. <span class="null-checkbox"></span>

The header checkbox works perfectly
